apps/docs/src/.vitepress/components/ComponentCard.stories.ts

```diff
@@ -0,0 +1,43 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import ComponentCard from "./ComponentCard.vue";
+
+const meta: Meta<typeof ComponentCard> = {
+  title: "roadmap/ComponentCard",
+  ...defineStorybookActionsAndVModels({
+    component: ComponentCard,
+    events: [],
+    decorators: [
+      (story) => ({
+        components: { story },
+        template: `<div style="width: 16rem;"> <story /> </div>`,
+      }),
+    ],
+  }),
+};
+
+export default meta;
+type Story = StoryObj<typeof ComponentCard>;
```

Comment:
please add an example with n/a due date 
![image](https://github.com/SchwarzIT/onyx/assets/151019977/21fc8a74-0745-4926-9de5-75e6d4311250)
in the storybook controls I can't find out how to "unset" the date control

Reply:
Good point :D Looks like Storybook does not provide a way to do this. Should I change the control to string?

Reply:
I added a story without a due date

Reply:
it's handy like the control is now, the extra story does the trick :p thx!